src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/test/TestBase.java

```diff
@@ -47,6 +49,10 @@
 	
 	public static final int MAX_SERVER_STARTUP_IN_MILLISECONDS = 10 * 60 * 1000;
 	
+	protected String firstPatientIdentifier;
```

Comment:
hello @sherrif10 , can you please put this variable where its supposed to be??

Reply:
This variable is supposed to be in TestBase class to be generic.

Reply:
what do you mean to be generic ??

Reply:
> what do you mean to be generic ??

I mean it method can be reusable